File: Lab0/suma.c
#include <stdio.h>
#include <stdlib.h>

/**
* @brief Suma de numeros recibidos por consola
*
*
*/

int suma(int argc, char** argv){
  double suma =0;
  int i=0;
  for (i=0; i<argc; i++){
    suma+=atof(argv[i]);
  }
  printf("%f\n", suma);
  return 0;
}

/**
* @brief Función main que llama a la función suma
*
*
*/

int main(int argc, char** argv){
  suma(argc,argv);
  return 0;
}

File: Lab0/suma.py
#!/usr/bin/python

import sys
"""biblioteca para habilitar funciones de la consola"""

def Suma():
    """funcion suma que recibe elementos desde la consola y los suma"""
    counter = 0
    """variable counter para guardar la suma de los elementos"""
    array_nums= sys.argv
    """arreglo que contiene lo que recibe la consola"""
    for elements in range(1,len(sys.argv)):
        """ciclos va desde el segundo elemento para eliminar ./suma.py hasta n cantidad de numeros ingresados"""
        counter = counter + float(array_nums[elements])
        """el ciclo recorre el arreglo, suma los elementos y los guarda en counter"""
    print counter
    """imprime el resultado"""
Suma()
"""para ejecutar la funcion automaticamente"""
